apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  IconButton,
+  NotificationMarker,
+  NotificationMarkerProps,
+} from '@itwin/itwinui-react';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
```

Comment:
**[DONE]** Nice idea, thanks

Reply:
Hm 🤔 I just thought about how there could be a problem with visual testing the pulse if one time it's 'on' and another time it's 'off'. Have you noticed a problem with that if you run that test a few times?

Reply:
I thought of that but then realized that the CSS repo also does a similar form of image testing. The [header](https://github.com/iTwin/iTwinUI/blob/main/packages/itwinui-css/backstop/results/bitmaps_reference/iTwinUI_dark_header_Type_All_-_narrow_screen_0_demo-all_0_.png) and [side-navigation](https://github.com/iTwin/iTwinUI/blob/main/packages/itwinui-css/backstop/results/bitmaps_reference/iTwinUI_dark_side-navigation_State_collapsed_hover_and_focus_0_demo-side-navigation_0_desktop.png) tests also have a NotificationMarker in the screenshot. Since it worked there with Backstop, I thought it might work here too with Cypress.

Additionally, I ran the tests a couple of times and they always passed. Also, there are multiple attempts for each test (around 4), so I think they will pass even if they fail in one of the attempts

@mayank99 What do you think?

Reply:
Good info, thanks!